tools/lint/sources.swiftlint.yml

```diff
@@ -30,7 +30,7 @@ whitelist_rules: # we enable lint rules explicitly - only the ones listed below
   - force_try
   - force_unwrapping
   - function_default_parameter_at_end
-  - identifier_name
+  # - identifier_name
```

Comment:
If we need to bypass some linter rules, no reason to disable it globally - Swiftlint has [Disable Rules in Code](https://github.com/realm/SwiftLint#disable-rules-in-code) feature in-build. Let's do either:
```swift
// swiftlint:disable identifier_name
let a: String = ...
// swiftlint:enable identifier_name
```
or:
```swift
let a: String = ... // swiftlint:disable:this identifier_name
```

Reply:
this is a very good point: when i search for `swiftlint:disable:this identifier_name` in the project, there are too many of them. if we need to disable this more than once, let's disable it for good.?
and `identifier_name` doesn't make the code safer unlike `forced_unwrapping` (which also has quite a few exceptions in code)

Reply:
OK, let's disable this 👌. We keep an eye on code verbosity in code reviews, so I'm OK to remove this rule. But please also update the code and delete all `// swiftlint:disable:this identifier_name` along with deleting (not disabling by `#` comment) the `- identifier_name` rule.